rootfs/api/views.py

```diff
@@ -152,7 +152,7 @@ def create(self, request, *args, **kwargs):
         try:
             return super(BaseDeisViewSet, self).create(request, *args, **kwargs)
         # If the scheduler oopsie'd
-        except RuntimeError as e:
+        except Exception as e:
```

Comment:
Debating if I should rather import `KubeException` here and use that if we are specifically wanting `scheduler` problems but I have a feeling this is more of a catch all scenario, which can include Django and other ones?